iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -321,6 +323,11 @@ public void configureJobConf(TableDesc tableDesc, JobConf jobConf) {
       jobConf.set(opKey, tableDesc.getProperties().getProperty(opKey));
       Preconditions.checkArgument(!tableName.contains(TABLE_NAME_SEPARATOR),
           "Can not handle table " + tableName + ". Its name contains '" + TABLE_NAME_SEPARATOR + "'");
+      if (tableDesc.getProperties()
+          .get(HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX + tableName) != null) {
```

Comment:
can we use static imports here?

Reply:
No we cant. It complains about some checkstyle error.

Reply:
HiveIcebergMetaHook has static imports and compiles fine

Reply:
The error - 
`[ERROR] /Users/sbadhya/apache/hive/iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java:212: Using a static member import should be avoided - org.apache.hadoop.hive.ql.security.authorization.HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX. [AvoidStaticImport]`